domain-models-interface-extensions/src/main/java/org/folio/rest/tools/utils/ClassPath.java

```diff
@@ -0,0 +1,547 @@
+package org.folio.rest.tools.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.jar.Attributes;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.collections4.MultiMapUtils;
+import org.apache.commons.collections4.SetValuedMap;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+
+public class ClassPath {
```

Comment:
Can you add the source where this has been copied from, the copyright attribution, and the license?

Reply:
That would be something @dmtkachenko presumably knows.

Reply:
I copied the licence from file that this is based upon (AFAIK)

Reply:
Yes, correct. It was originally copied and modified from
https://github.com/google/guava/blob/v29.0/guava/src/com/google/common/reflect/ClassPath.java

Reply:
This copy still violates the requirements of the Apache License.

Reply:
How does it violate the License requirements?

Reply:
In what way? ClassPath is Apache 2.0. So is RMB

Reply:
If changed the notice to comply with this license requirement: "You must cause any modified files to carry prominent notices stating that You changed the files"